Create Blog “2025/05/07/monthly-update-for-april-2025” (#5965)
Couchers-org/couchers: app/web/markdown/blog/2025/05/07/monthly-update-for-april-2025.md

---
title: Monthly Update for April 2025
slug: monthly-update-april-2025
description: Monthly post outlining new features we have implemented and what
  has changed in April 2025.
date: 2025/05/07
author: ChrisK
share_image: /app/web/public/img/blog/20210825_talkingtown.png
---

> Buckle up for our monthly development update! Read about the new features we have implemented and what has changed in April 2025.

Reviewer: ```suggestion
Buckle up for our monthly development update! Read about the new features and changes in April 2025.
```

> **This has been added:**

Reviewer: ```suggestion
**The following are new additions:**
```

> * A **Notifications Feed** so you don’t miss any important messages (by [Nicole](https://couchers.org/user/unsettleddown))

Reviewer: ```suggestion
* The **search results list view** on the map page lets you see many search results at once (by [Nicole](https://couchers.org/user/unsettleddown))
* **New search filters** have been added to allow you to easily filter inactive and empty profiles, or search for people who are keen to meet up (by [Nicole](https://couchers.org/user/unsettleddown))
* A **Notifications Feed** so you don’t miss any important messages (by [Nicole](https://couchers.org/user/unsettleddown))
```

* A **Donations Portal** to easily manage and review your donations (by [Aapeli](https://couchers.org/user/aapeli))
* **Strong Verification** has been released into account settings after testing in feature preview (by [Aapeli](https://couchers.org/user/aapeli))

> * **Username link** on the profile to easily share it (by [Pablo](https://couchers.org/user/pcolt86))

Reviewer: ```suggestion
* **Username & profile link** on profile pages to easily share profiles (by [Pablo](https://couchers.org/user/pcolt86))
```

**These things have changed:**

> * Complete overhaul of the **Map Search** and improved filter options to find who you are looking for in a snap (by [Nicole](https://couchers.org/user/unsettleddown))

Reviewer: ```suggestion
* Complete overhaul of the **Map Search** which now shows the total number of results found (and how many are currently displayed). You can also expand the search results area & minimize the map so you can see more results at once. There are also improved filter options to find who you are looking for in a snap (by [Nicole](https://couchers.org/user/unsettleddown))
```

> * New **Roadmap design** to easily see where couchers is headed and how far we’ve gotten already (by [Chris](https://couchers.org/user/chrisk))

Reviewer: ```suggestion
* New **Roadmap design** to easily see what we've been working on and what's coming next (by [Chris](https://couchers.org/user/chrisk))
```

* Improved **Event Time Picker**, it doesn’t just look better, it’s also easier to use (by [Colleen](https://couchers.org/user/colleen))

> * Started the **Blog Revamp** to keep you better informed (by [Aapeli](https://couchers.org/user/aapeli), [Chris](https://couchers.org/user/chrisk), [Colleen](https://couchers.org/user/colleen))

Reviewer: ```suggestion
* Started the **Blog Revamp** to keep you better informed (by [Aapeli](https://couchers.org/user/aapeli), [Chris](https://couchers.org/user/chrisk), [Colleen](https://couchers.org/user/colleen))
* Updated and added more **[Help Center](https://help.couchers.org) pages** to provide clarity and answer more frequently asked questions (by [Jesse](https://couchers.org/user/jesse))
```

## Development stats

**In April 2025 Couchers.org had:**

* **15 Developers** contribute in
* **209 Changes** that changed
* **283 Files** with **6068 Additions** and **3670 Deletions**.

None of this would be possible without our volunteers working in all areas of the project. The development team would like to thank the rest of the contributors for making our work possible.

> We will keep these updates short and concise, you can find in depth information on our [github project boards](https://github.com/orgs/Couchers-org/projects).

Reviewer: ```suggestion
We will keep these updates short and sweet, but you can find more detailed/technical information on our [github project boards](https://github.com/orgs/Couchers-org/projects).
```

## Think we could do better? 

Help us improve Couchers!

**Developers:**

> Please join our team on [GitHub](https://github.com/Couchers-org) or just give us a star if you appreciate our work! All our code is open source and we need all the help we can get in building the best Couchsurfing platform.

Reviewer: ```suggestion
Please join our team on [GitHub](https://github.com/Couchers-org/couchers) or just give us a star if you appreciate our work! All our code is open source and we need all the help we can get in building this modern, non-profit, community-driven couchsurfing platform.
```

> **Bloggers and Volunteers:**

Reviewer: ```suggestion
**Bloggers, Marketing Professionals, Designers, & Other Volunteers:**
```

> **[Sign up](https://couchers.org/volunteer)** and assist us in our endeavor.

Reviewer: ```suggestion
**[Sign up](https://couchers.org/volunteer)** and assist us in our endeavors.
```

**Donors:**

> Your [donations](https://couchers.org/donate) are highly appreciated, see how we use them [here](https://couchers.org/blog/2025/04/20/couchers-inc-financials-2024).

Reviewer: ```suggestion
We would appreciate a [donation](https://couchers.org/donate) from you. Donations are essential for keeping the platform running since we are a 501c3 non-profit. Check out or financials [here](https://couchers.org/blog/2025/04/20/couchers-inc-financials-2024).
```

## Don’t want to commit?

> Consider hosting and joining events in your city or on your travels and spread the word. Help us grow this community to reach our next milestone of 100 000 Members.

Reviewer: ```suggestion
Host or join events in your city or while traveling, and help spread the word. Let’s grow this community together and reach our next milestone of 100,000 members!
```

**Bloggers, Marketing Professionals, Designers, & Other Volunteers:**

**[Sign up](https://couchers.org/volunteer)** and assist us in our endeavors.

**Donors:**

We would appreciate a [donation](https://couchers.org/donate) from you. Donations are essential for keeping the platform running since we are a 501c3 non-profit. Check out or financials [here](https://couchers.org/blog/2025/04/20/couchers-inc-financials-2024).

## Don’t want to commit?

Host or join events in your city or while traveling, and help spread the word. Let’s grow this community together and reach our next milestone of 100,000 members!

[Volunteers ](https://couchers-git-cms-generalroadmap-couchers-org.vercel.app/team)and [donations](https://couchers.org/blog/2025/04/20/couchers-inc-financials-2024) are what make Couchers.org possible!

<a href="https://www.instagram.com/couchersorg/#" target="_blank"><img src="/img/blog/instagram_logo.svg" alt="IG" style="width:26px !important;" /></a> <a href="https://bsky.app/profile/couchers.bsky.social/" target="_blank"><img src="/img/blog/bluesky_logo.svg" alt="bsky" style="width:26px !important;" /></a>

> *Written by `Chris`[](https://couchers.org/user/chrisk). Published on 2025/05/10*

Author: ```suggestion
*Written by [`Chris`](https://couchers.org/user/chrisk). Published on 2025/05/10*
```
